package/src/skia/Skia.ts

```diff
@@ -27,6 +27,32 @@ import type { SkRSXform } from "./RSXform";
 import type { SkPath } from "./Path/Path";
 import type { SkContourMeasureIter } from "./ContourMeasure";
 import type { PictureFactory, SkPictureRecorder } from "./Picture";
+import type { Color, SkColor } from "./Color";
+
+/*
+ * Parse CSS colors
+ */
+const SkiaColor = (cl: Color) => {
+  if (typeof cl === "number") {
+    return cl;
+  }
+  let color = Skia.parseColorString(cl);
+  if (color === undefined) {
+    console.warn("Skia couldn't parse the following color " + cl);
```

Comment:
Should we add an explanation here - like "This might affect your drawing performance since the colour will be evaluated in Javascript instead of on the native side at runtime"